cerbero/build/recipe.py

```diff
@@ -127,6 +127,7 @@ class BuildSteps(object):
 
     FETCH = (N_('Fetch'), 'fetch')
     EXTRACT = (N_('Extract'), 'extract')
+    CLEAN_OLD = (N_('Clean old'), 'clean_old')
```

Comment:
I would have call this clean_old_files or clean_obsolete_files because clean_old only is not super explicit ;)

Reply:
That makes sense and it's easily tackled. I'll rename it to `clean_old_files` then